globals.go

```diff
@@ -94,16 +104,13 @@ func (r *rawGlobals) eval(meta StackMetadata) (*Globals, error) {
 		return nil, err
 	}
 
+	globals := newGlobals()
 	// error messages improve if globals is empty instead of undefined
```

Comment:
maybe now this comment can be removed?

Reply:
actually no, it is a little odd to add an empty globals to the ctx, and that is done just to improve error messages (doesn't change behavior). I mean...we could remove the comment anyway...but since the "why" is kinda funny on this case I went with the comment.